### ISSUE TITLE
Jakarta EE API (10.0.0) has an old activation framework dependency

### ISSUE THREAD

**Issue by lprimak** [eclipse-ee4j/jakartaee-api#134]
Jakarta EE API artifact depends on an old (version 1.2.2) of `activation-api`
Currently, this causes no issues since this old version uses `javax` package

I believe that Jakarta EE API should not depend on any of the SPEC APIs as reported in the related issue.
This is just another symptom of this bug.

Relates to #133 
```
[INFO] +- jakarta.platform:jakarta.jakartaee-api:jar:10.0.0:provided
[INFO] |  +- jakarta.platform:jakarta.jakartaee-web-api:jar:10.0.0:provided
...
[INFO] |  |  +- jakarta.authentication:jakarta.authentication-api:jar:3.0.0:provided
[INFO] |  |  \- jakarta.security.enterprise:jakarta.security.enterprise-api:jar:3.0.0:provided
[INFO] |  +- jakarta.activation:jakarta.activation-api:jar:1.2.2:provided
```

**Comment by pzygielo** [comment]
Cannot confirm
```
$ wget https://repo.maven.apache.org/maven2/jakarta/platform/jakarta.jakartaee-api/10.0.0/jakarta.jakartaee-api-10.0.0.pom
$ 
$ mvn -f jakarta.jakartaee-api-10.0.0.pom dependency:tree
...
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ jakarta.jakartaee-api ---
[INFO] jakarta.platform:jakarta.jakartaee-api:jar:10.0.0
[INFO] +- jakarta.platform:jakarta.jakartaee-web-api:jar:10.0.0:compile
...
[INFO] |  +- jakarta.authentication:jakarta.authentication-api:jar:3.0.0:compile
[INFO] |  \- jakarta.security.enterprise:jakarta.security.enterprise-api:jar:3.0.0:compile
[INFO] +- jakarta.jms:jakarta.jms-api:jar:3.1.0:compile
[INFO] +- jakarta.activation:jakarta.activation-api:jar:2.1.0:compile
[INFO] +- jakarta.mail:jakarta.mail-api:jar:2.1.0:compile
...
[INFO] \- org.glassfish:jakarta.faces:jar:4.0.0:provided
...
$ 
$ grep -A4 -B2 jakarta.activation-api jakarta.jakartaee-api-10.0.0.pom 
    <dependency>
      <groupId>jakarta.activation</groupId>
      <artifactId>jakarta.activation-api</artifactId>
      <version>2.1.0</version>
      <scope>compile</scope>
      <optional>false</optional>
    </dependency>
```

**Comment by lprimak** [comment]
@pzygielo you are correct.
I had a bom that downgraded activation-api that was causing this.
However, this is another reason to fix #133 which is very much still valid.
Thanks for the repdocucer!
